runtime/lua/tohtml.lua

```diff
@@ -0,0 +1,1263 @@
+---@defgroup vim.tohtml
+
+-- The HTML conversion script is different from Vim's one. If you want to use
+-- Vim's TOhtml converter, download it from the vim GitHub repo.
+-- Here are the Vim files related to this functionality:
+-- - https://github.com/vim/vim/blob/master/runtime/syntax/2html.vim
+-- - https://github.com/vim/vim/blob/master/runtime/autoload/tohtml.vim
+-- - https://github.com/vim/vim/blob/master/runtime/plugin/tohtml.vim
+--
+-- Main differences between this and the vim version:
+-- - No "ignore some visual thing" settings (just set the right Vim option)
+-- - No support for legacy web engines
+-- - No support for legacy encoding (supports only UTF-8)
+-- - No interactive webpage
+-- - No specifying the internal html (no XHTML, no use_css=false)
+-- - No multiwindow diffs
+-- - No ranges
+--
+-- Remarks:
+-- - Not all visuals are supported, so it may differ.
+
+--- @class vim.tohtml.opt
+--- @field title? string|false
+--- @field number_lines? boolean
+--- @field font? string[]|string
+--- @field width? number
```

Comment:
```suggestion
--- @field width? integer
```

Always use `integer` where is makes sense.